source/renderer/app/features/discreet-mode/__mocks__/react-svg-inline.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import React from 'react';
+
+export default function ReactInlineSVG() {
```

Comment:
Do we need this? is not used anywhere.

Reply:
Yes, it is used for integration tests: https://github.com/input-output-hk/daedalus/pull/2756